bundle/edu.gemini.model.p1/src/main/xsd/Target.xsd

```diff
@@ -61,6 +61,7 @@
                 </xsd:sequence>
                 <xsd:attribute name="epoch" type="CoordinatesEpoch" default="J2000"/>
                 <xsd:attribute name="horizonsDesignation" type="xsd:token"/>
+                <xsd:attribute name="horizonsQuery" type="xsd:token"/>
```

Comment:
`horizonsQuery` is optional so it shouldn't require an upconverter. It's derivable from `horizonsDesignation` (and vice-versa) but these conversions are unavailable to the PDF generator (the only place it's used), so instead we have a redundant attribute.